docs/getting_started.rst

```diff
@@ -53,19 +44,19 @@ Once that's finished building, you can source the new colcon overlay:
 
     source install/setup.bash
 
-**Note**: the sourcing of the local colcon overlay is required every time the colcon workspace is opened in
+**Note**: The sourcing of the local colcon overlay is required every time the colcon workspace is opened in
 a new shell environment.
 As an alternative, you can copy the source command with the full path of your local installation to your 
 :code:`.bashrc` file as:
 
 .. code-block:: bash
 
-    source ~/PATH_TO_WORKSPACE/workspaces/is/install/setup.bash
+    source ~/PATH_TO_WORKSPACE/is-workspace/install/setup.bash
```

Comment:
If `PATH_TO_WORKSPACE` is the "full path" to the local, this line should be:

    source PATH_TO_WORKSPACE/is-workspace/install/setup.bash

Remember that `~` means "current user home directory".

Reply:
ok.